dingo/gw/training/train_pipeline.py

```diff
@@ -21,9 +27,57 @@
     build_train_and_test_loaders,
 )
 from dingo.core.utils.trainutils import EarlyStopping
+from gw.dataset import WaveformDataset
+from core.posterior_models import BasePosteriorModel
```

Comment:
```suggestion
from dingo.gw.dataset import WaveformDataset
from dingo.core.posterior_models import BasePosteriorModel
```
This looks like a typo. Not sure how it got here...

Reply:
Fixed in latest commit.